Curs11/C/01-copy_text.c

```diff
@@ -0,0 +1,59 @@
+/* Programarea Calculatoarelor CA, 2022-2023
+
+	Program care realizeaza copierea unui fisier text.
+
+*/
+
+#include <stdio.h>
+
+int main()
+{
+	FILE *copyFrom, *copyTo;
+	char filename[101];
+	int read = 0;
+
+	printf("File to copy: ");
+	scanf("%100s", filename);
+
+	if ((copyFrom = fopen(filename, "r")) == NULL)
```

Comment:
do no open file in if statements.

Reply:
Opening like this was a suggestion that @florinpop1981 made.

Reply:
@RaduNichita

Reply:
```cpp
FILE* copyFrom = fopen(filename, "r");
if (!copyFrom) {
    ...
}
```